pkg/agent/clientset.go

```diff
@@ -24,10 +24,20 @@ import (
 
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/connectivity"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/klog/v2"
 )
 
+const (
+	connectedPodConditionType      = "k8s.io/apiserver-network-proxy-connected"
+	connectedPodConditionReason    = "Connected"
+	disconnectedPodConditionReason = "Disconnected"
+	partitionPodConditionReason    = "Partition"
```

Comment:
Not 100% on this terminology. Maybe `PartiallyConnected`?

Reply:
I'd prefer something like PartiallyConnected. "partition" suggests a split brain problem and we (ANP agent) are not in a position to detect that. (Actually we almost might be but that is certainly not what this condition is indicating)

Reply:
Agreed that partition is a misnomer here. Changed to PartiallyConnected.